lib/inertia_rails/configuration.rb

```diff
@@ -9,6 +9,9 @@ class Configuration
       # Overrides Rails default rendering behavior to render using Inertia by default.
       default_render: false,
 
+      # Allows the user to hook into the default rendering behavior and change it to fit their needs
+      component_path_resolver: ->(path, action) { "#{path}/#{action}" },
```

Comment:
Another quick idea (srr 😅): should we use props instead of args — that way we can make the interface less fragile: `->(path:, action:) { "#{path}/#{action}" }`. Say, in case we decide to pass controller in the future (if someone comes with a good case for that).

Reply:
Good call! Could save us from having to bump a major version in the future.